languages/gutenberg.pot

```diff
@@ -3,15 +3,15 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "X-Generator: babel-plugin-wp-i18n\n"
 
-#: editor/blocks/freeform/index.js:4
```

Comment:
Actually, on the topic of localization, we should localize the block's title.

See text bock as example:

https://github.com/WordPress/gutenberg/blob/f1b8d8d/editor/blocks/text/index.js#L5